src/components/AddView.js

```diff
@@ -1,10 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
 
 const AddView = () => {
+  const [state, setState] = useState({
+    purchaseItem: '',
+    howSoon: '7',
+    lastPurchaseDate: null,
+  });
+
+  const handleChange = (e) => {
+    setState({
+      ...state,
+      [e.target.name]: e.target.value,
```

Comment:
I’m not sure this is correct. I think you could extend state and set a purchase item to be 
e.target.value like this: ```setState({ ...state, purchaseItem: e.target.value })```

Reply:
Hey @katia-utochkina , we not updating the purchase Item only but also the howSoon field. Hence that code covers for both fields

Reply:
Also, @ciradu2204 , you've done a really-really good job